### PR TITLE
Update CasperJS example

### PR DESCRIPTION
**Pull request by chekunkov**
- Use base image.
- Rearrange scripts.
- Move Scrapinghub specific utils to a separate module.
- jq is not needed any more.

TODO

- [x] Rename base image to scrapinghub-image-casperjs

### CODE REVIEW

File: examples/javascript/casperjs/app/spiders/query-casperjs.js [hunk 1]
```diff
@@ -1,18 +1,20 @@
 // Adapted example from http://casperjs.org/
-var system = require('system');
-var fs = require('fs');
-var fifopath = system.env.SHUB_FIFO_PATH;
-var fifo = fifopath ? fs.open(fifopath, 'a+') : system.stdout;
+var shubUtils = require('../shub-utils');
+
+var fifo = shubUtils.getFifo();
 
 function writeCmd(command, payload) {
```
Comment (chekunkov) [inline examples/javascript/casperjs/app/spiders/query-casperjs.js]: hum, maybe move to shub-utils.js as well?

Comment (redapple) [inline examples/javascript/casperjs/app/spiders/query-casperjs.js]: Actually, `query-casperjs.js` was about making minimal changes to the example found on http://casperjs.org/

**Comment by chekunkov** [comment]
Base image is https://github.com/scrapinghub/scrapinghub-image-casperjs/releases/tag/0.0.3 now. Removed shub-utils.js - job arguments are passed via command line options. `env2options` script can be used to pass job settings the same way.

File: examples/javascript/casperjs/scripts/start-crawl [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -e
+
+SCRIPT="/app/$SHUB_SPIDER"
+SCRIPT_ARGS=$(env2options arguments)
```
Comment (redapple) [inline examples/javascript/casperjs/scripts/start-crawl]: what is `env2options`?

Comment (chekunkov) [inline examples/javascript/casperjs/scripts/start-crawl]: check base image repo https://github.com/scrapinghub/scrapinghub-image-casperjs#env2options

**Comment by chekunkov** [comment]
@vshlapakov please update base image after https://github.com/scrapinghub/scrapinghub-image-casperjs/pull/2 is merged and squash the changes before merge

**Pull request merged**